src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2155,13 +2105,13 @@ public override string GetName(SyntaxNode declaration)
                 case SyntaxKind.MethodDeclaration:
                     return ((MethodDeclarationSyntax)declaration).Identifier.ValueText;
                 case SyntaxKind.FieldDeclaration:
-                    return GetName(((FieldDeclarationSyntax)declaration).Declaration);
+                    return this.GetName(((FieldDeclarationSyntax)declaration).Declaration);
```

Comment:
Now, i personally prefer 'this' most of the time because it makes it completely clear if something is local or instance state.  And it also makes it clear if it's instance or static.  Given that we don't do things like S_MethodName, i like just having the bit of syntax to make it clear.